kani-driver/src/call_cbmc.rs

```diff
@@ -45,7 +45,7 @@ impl KaniSession {
         } else {
             // Spawn the CBMC process and process its output below
             let cbmc_process = self.run_piped(cmd)?;
-            if cbmc_process.is_some() {
+            if let Some(..) = cbmc_process {
```

Comment:
I think the point of the clippy warning is to get rid of the `.unwrap()`. So the code should be `if let Some(cbmc_process) = cbmc_process`. Then you can get rid of the unwrap below.

Reply:
Right, thanks! These should all be fixed now, please let me know if you spot others :D